pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -355,10 +296,9 @@ stdenv.mkDerivation (rec {
   '';
 
   passthru = {
-    inherit version updateScript;
+    inherit updateScript;
+    version = ffversion;
```

Comment:
I am tempted to propose changing `ffversion` back to `version`. I was never really happy about that change but it made sense when we shared the expression with other browser flavors. Opinions?

Reply:
done in an additional commit.

Reply:
Clearly @oxij has opinions about this point. I am fine with leaving it in here. I asked for opinions for a reason.

Maybe we should indeed revert this, if only, to show good intend and not cause further harm. After all it doesn't really mean much to me (at least).